app/screens/post_options/options/follow_option.tsx

```diff
@@ -40,8 +42,15 @@ const FollowThreadOption = ({thread}: FollowThreadOptionProps) => {
         }
     }
 
+    const unFollowThread = useCallback(() => {
```

Comment:
@matthewbirtch : When we follow a thread, the snackbar shows up with the `undo` action.  But now, on long pressing on the thread, we have the `unfollow` option.  Should we be displaying the snackbar again if the user presses on the `unfollow` button?

Reply:
@avinashlng1080 I don't think we should show the snackbar when you follow a thread - only when you `unfollow` a thread.

Reply:
I would say that they snackbar is not needed for either one as there is UI changes in the channel or thread screen already